src/appengine/libs/issue_filer.py

```diff
@@ -222,15 +173,29 @@ def file_issue(testcase,
   if testcase.uploader_email and testcase.uploader_email not in ccs:
     ccs.append(testcase.uploader_email)
 
-  if issue_tracker.project == 'oss-fuzz' and ccs:
-    # Add a reported label for deadline tracking.
-    issue.labels.add(reported_label())
+  ccs.extend(properties.ccs)
+
+  # Get view restriction rules for the job.
+  issue_restrictions = data_handler.get_value_from_job_definition(
+      testcase.job_type, 'ISSUE_VIEW_RESTRICTIONS', 'security')
```

Comment:
maybe add define ISSUE_VIEW_RESTRICTIONS as security in env.yaml and give both all and security strings in comment ?